packages/ui/src/components/organisms/Form/DynamicForm/validators/document/types.ts

```
@@ -0,0 +1 @@
+export interface IDocumentValidatorParams {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)